run-tests.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+ES_VER=0.90.13
+ES_TMP=/tmp/serviced_elastic
+ES_DIR=$ES_TMP/elasticsearch-$ES_VER
+
+if [[ -z $GORACE ]]; then
+	export GORACE="history_size=7,halt_on_error=1";
```

Comment:
Does comma separator work?  According to the [doc](https://golang.org/doc/articles/race_detector.html#Options), separate options are supposed to be separated by spaces, which is what the makefile in an earlier revision